coriolis/osmorphing/windows.py

```diff
@@ -433,8 +467,89 @@ def _install_cloudbase_init(self, download_url,
 
         return cloudbaseinit_base_dir
 
+    def _compile_static_ip_conf_from_registry(self, key_name):
+        ips_info = []
+        interfaces_reg_path = INTERFACES_PATH_FORMAT % key_name
+        interfaces = self._conn.exec_ps_command(
+            "(((Get-ChildItem -Path '%s').Name | Select-String -Pattern "
```

Comment:
Please confirm that all of these registry keys exist on the oldest currently supported Windows (2012R2) as well as the latest one by running a test of each so as to cover all bases.